cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,221 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-8872` - Changed the Apache Tephra dependency to the 0.11.0-incubating
+  version.
+
+- :cask-issue:`CDAP-9250` - Added support for the Hive version shipped with HDP 2.6.
+
+- :cask-issue:`CDAP-7731` - Added fetch size and transaction flush interval configurations
+  to the Kafka Consumer Flowlet.
+
+- :cask-issue:`CDAP-8902` - Added support for adding, deleting, updating, and retrieving
+  workflow schedules in the CDAP CLI.
+
+- :cask-issue:`CDAP-9117` - Changed the log viewer to only show "user" condensed logs by
+  default. There is an option to "Include System Logs" to show all logs as previously.
+
+- :cask-issue:`CDAP-9141` - Added a 'sparkprogram' plugin type that can be used to run
+  arbitrary Spark code at the start or end of a pipeline.
+
+- :cask-issue:`CDAP-9147` - Added pipeline start and finish logs in the condensed view.
+
+- :cask-issue:`CDAP-9235` - Added workspace tabs to the top panel of the Data Prep plugin.
+
+- :cask-issue:`CDAP-9281` - Added support for CDH 5.11.0.
+
+- :cask-issue:`CDAP-9327` - Added support in the CDAP UI for the Spark program pipeline
+  plugin.
+
+- :cask-issue:`HYDRATOR-1316` - Added the ability to see logs in preview.
+
+- :cask-issue:`WRANGLER-51` - Added Copy Column to the list of column dropdown directives
+  in the Data Prep plugin.
+
+- :cask-issue:`WRANGLER-77` - Added a column-action dropdown menu for Data Preparation
+  directives.
+
+Improvements
+------------
+
+- :cask-issue:`CDAP-7352` - Improved log viewer to always show headers at the top of the
+  screen, even when the user scrolls down.
+
+- :cask-issue:`CDAP-8430` - Modified the Entity list page to show appropriate message when
+  no entity is shown on the page, appropriate to how the user got to that state.
+
+- :cask-issue:`CDAP-8942` - Allow administrators to enable or disable updating schedules
+  using the property "app.deploy.update.schedules" in cdap-site.xml. Users can override this
+  to enable or disable updating schedules during deployment of an application using the same
+  property specified in the configuration of application.
+
+- :cask-issue:`CDAP-8982` - Updated SDK images (Amazon AMI, Docker image, Virtual Machine
+  image) to use Ubuntu 16.04 LTS release.
+
+- :cask-issue:`CDAP-8990` - New configurations have been added to control the YARN
+  application master container memory size, maximum heap memory size, and maximum non-heap
+  memory size: twill.java.heap.memory.ratio, twill.yarn.am.memory.mb, and
+  twill.yarn.am.reserved.memory.mb.
+
+- :cask-issue:`CDAP-9003` - Increased the memory allocation for the CDAP Explore service
+  container to 2048MB.
+
+- :cask-issue:`CDAP-9017` - Simplified the status, next runtime of pipelines, total number
+  of running pipelines, and drafts in the pipeline list view UI.
+
+- :cask-issue:`CDAP-9070` - Added the ability to work with Data Preparation from within
+  the CDAP Studio UI. Users can now open a wrangler transform, wrangle the data, and save it
```

Comment:
I have change "Data Prep" to "Data Preparation", and "Data Preparation" to "Data Preparation (Wrangler)" when talking about the plugin.